### PR TITLE
Update ReadMes based on feedback

### PR DESCRIPTION
**Pull request by ehoops-microsoft**
# Purpose of PR

Update lab readmes based on feedback from Accenture in #52 

## Type of PR

- [x] Documentation changes
- [ ] Labs changes
- [ ] Codespaces changes
- [ ] Code changes
- [ ] Test changes

### CODE REVIEW

File: labs/beta-labs/res-edge-labs/deploy-res-edge/README.md [hunk 1]
```diff
@@ -218,15 +219,40 @@ kic logs resedge
 
 ```
 
-## Observability: K9s
+## Observability
```
Comment (bartr) [inline labs/beta-labs/res-edge-labs/deploy-res-edge/README.md:222]: I like this!

**Review by bartr — APPROVED** [review]
a couple of comments - looks great!

File: labs/beta-labs/res-edge-labs/deploy-res-edge/README.md [hunk 2]
```diff
@@ -26,11 +26,12 @@ kic cluster create
 
 kic pods
 
-# wait for pods to get to Running
+# wait for pods to get to Running 1/1
```
[the remark this answers is not in the document]
Comment (ehoops-microsoft) [reply labs/beta-labs/res-edge-labs/deploy-res-edge/README.md:29]: Yep switched them around!

File: labs/beta-labs/res-edge-labs/deploy-res-edge/README.md [hunk 3]
```diff
@@ -26,11 +26,12 @@ kic cluster create
 
 kic pods
 
-# wait for pods to get to Running
+# wait for pods to get to 1/1 Running
 # ctl-c to exit
 kic pods --watch
 
 ```
+> The `--watch` flag will update the last line in the terminal output. Wait for the last line to read 'Running' after the 'ContainerCreating' status then `ctrl-c` to exit
```
Comment (madossan01) [inline labs/beta-labs/res-edge-labs/deploy-res-edge/README.md:34]: Suggestion to replace "last line" "line with the cluster pod you just created"

**Pull request merged**